Whisper.net/WhisperAlignmentHeadsPreset.cs

```diff
@@ -0,0 +1,21 @@
+// Licensed under the MIT license: https://opensource.org/licenses/MIT
+
+public enum WhisperAlignmentHeadsPreset
```

Comment:
In general I tried to "mask" the Internal enums and the public enums and apply some mapping so that we can evolve and keep backwards compatibility even if the native whisper.cpp library will remove/add new options.  (e.g. [Managed WhisperLogLevel vs native GgmlLogLevel](https://github.com/sandrohanea/whisper.net/blob/146f50c1367d360876cbab769ddc44cc8aafb692/Whisper.net/Logger/LogProvider.cs#L63) ) Maybe we can do the same to ensure better flexibility for us in the future.